pep-0606.rst

```diff
@@ -0,0 +1,223 @@
+Marking APIs as provisional offers a way to avoid that constraint, but actually
+taking advantage of that freedom causes other problems.
+
+PEP 602 doesn't attempt to address this problem.
+
+PEP 605 proposes to address it by actively seeking to promote adoption of
+CPython pre-releases for running production workloads, not just for library
+and application compatibility testing.
+
+(Note: some standard library APIs are amenable to initially being shipped as
+part of separately versioned packages via PyPI, and only later incorporated
+into the standard library. This section is more about the lower level APIs
+where that approach doesn't apply)
+
+
+Risks to be mitigated
+=====================
+
+While the status quo could stand to be improved in some respects, Python's
+popularity indicates that a lot of users and other participants in the wider
+Python ecosystem are happy enough with the current release management process.
+
+Python's user base is too large and
+`too varied <https://www.curiousefficiency.org/posts/2017/10/considering-pythons-target-audience.html>`__
+to cover all the potential downsides of changing our release cadence here, so
+instead this section just covers some of the points that have been specifically
+taken into account in the design of the PEPs.
+
+
+Impact on users and redistributors that already skip some releases
+------------------------------------------------------------------
+
+It is already the case that not all users and redistributors update to every
+published CPython release series (for example, Debian stable and Ubuntu LTS
+sometimes skip releases due to the mismatch between their 24 month release
+cycles and CPython's typically 18-month cycle).
+
+The faster 12-month full release cadence in PEP 602 means that users in this
+category may end up skipping two releases where they would previously have only
+skipped one. However, the extended notice period for deprecations means that
+skipping a single release should no longer result in missed deprecation warnings.
+
+With a 12-month release cadence each release branch will spend proportionally
+more time in security-fix-only mode (~48 months out of ~60 vs the current ~42
+out of ~60).
+
+The slower 24-month full release cadence in PEP 605 may move some of the users
+that have historically been in this category into the "update to every stable
+release" category.
+
+With a 24-month release cadence each release branch will spend proportionally
+less time in security-fix-only mode (~36 months out of ~60 vs the current ~42
+out of ~60).
+
+
+Impact on users and redistributors that update to every release
+---------------------------------------------------------------
+
+Many of Python's users never install a pre-release, but do update to every
+stable release series at some point after it is published.
+
+PEP 602 aims to mitigate the potential negative impact on members of this group
+by keeping the minimum gap between releases to 12 months.
```

Comment:
@ambv should tweak this to sell PEP 602 a little better :)

Reply:
This is one of the paragraphs I amended after you reminded me that PEP 602 had been updated to retain the 18 month support period.